apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -203,23 +222,34 @@ export class DeletionRequestUc {
 	}
 
 	private async removeUserFromTeams(deletionRequest: DeletionRequest) {
+		this.logger.debug({ action: ' removeUserFromTeams', deletionRequest });
```

Comment:
Only a question: 
The debug logs are always not visible if it is executed on production. The info for now also not. But this will be changed in future. Do you sure that this information should only displayed on local enviroments?

Reply:
@WojciechGrancow

Reply:
this is for local debug, we register in db action from data deletion in each module

Reply:
ok